jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/XGBoost.java

```diff
@@ -61,6 +61,20 @@ public static Booster loadModel(InputStream in)
     return Booster.loadModel(in);
```

Comment:
maybe we can add the logic converting InputStream into byte[] here

Reply:
+1 on this